docs/introduction/javascript-events-dom-apis.md

```diff
@@ -502,6 +502,41 @@ entityEl.addEventListener('physicscollided', collisionHandler);
 entityEl.removeEventListener('physicscollided', collisionHandler);
 ```
 
+### Binding Event Listeners
+
+By default, [Javascript execution context rules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this) binds `this` to the global context (`window`) for any independent function, meaning that these functions won't have access to the component's `this` by default.
+
+In order for the component's `this` to be accessible inside an event listener, [it must be bound](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this).
+
+There are several ways you can do this:
+
+1. By using an arrow function to define the event listener.  Arrow functions automatically bind `this`.
+
+```entityEl.addEventListener('physicscollided', function (event) {
```

Comment:
This example might be a bit convoluted. Attaching an event listener within an event listener. Perhaps we should simplify or add more context of what this example illustrates. I myself got confused.

Reply:
I don't understand why you describe this as "adding an event listener within an event listener".

This just shows addign an event listener that makes use of "this" inside the event listener (makign a console log, which is about the simplest thing I could think to do.

Maybe `this.el.addEventListener` is beter than `entityEl.addEventListener` as `this.el` is always defined and it may not be obvious what `entityEl` is...?

Reply:
RIght - have been looking at / editing this in Typora, where it looked like this:

![image](https://user-images.githubusercontent.com/16045703/150534141-53ec3e18-90e2-4b2c-8d80-e47e7cf2ee48.png)

Just looked at this in "source code" mode, and I see there is a load of extra code in there that definitely shouldn't be there.  (looks like this is because I had failed to include a newline after the ``` in MD).

Have fixed this up...